sys-kernel/buildkernel/buildkernel-1.0.37.ebuild

```diff
@@ -8,7 +8,7 @@ inherit eutils
 DESCRIPTION="Build secure boot EFI kernel with LUKS, LVM and plymouth"
 BASE_SERVER_URI="https://github.com/GenPi64"
 HOMEPAGE="${BASE_SERVER_URI}/${PN}"
-SRC_URI="${BASE_SERVER_URI}/${PN}/releases/download/${PV}/${P}.tar.gz"
+SRC_URI="${BASE_SERVER_URI}/${PN}/archive/refs/tags/${PV}.tar.gz"
```

Comment:
This is why I put this on the MR 
` ** This would prob be cleaner if the buildkernel-1.0.37.tar.gz existed at correct path, but this is how I got it to work for now **`

Maybe you can create a proper release for buildkernel repo you have and then I believe the original `SCR_URI`  would make more sense and prob work

![image](https://user-images.githubusercontent.com/400979/225704036-a2cad5bb-f908-4ae9-a462-4503f09865b7.png)

Reply:
I have no idea how it was originally made under Sakaki's repo so I cannot really replicate it so will you do the change or not?

Reply:
Since this is your repo, you have to make this change, or whoever else has maintainer access to the repo 

https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository?tool=webui

Reply:
I meant the original change regarding `-> ${PN}` in the ebuild.